price-alert-script/price_alert_script.py

```diff
@@ -14,6 +14,8 @@
 logging.basicConfig(filename='price_alert_logs.log', level=logging.INFO,
                     format='%(asctime)s - %(levelname)s - %(message)s')
 
+EMAIL_SENDER = 'trainee.tayla.dawson@sigmalabs.co.uk'
+
```

Comment:
Nice global constant.

Reply:
Maybe put this in an env file though if you don't want your email address public?